src/config/kernel.md

```diff
@@ -3,16 +3,21 @@
 ## Kernel series
 
 Void Linux provides many kernel series in the default repository. These are
-named `linuxX.X`, for example `linux4.19`. You can query for all available
+named `linuxX.X`: for example, `linux4.19`. You can query for all available
```

Comment:
I'm always unsure. Should we prefer `linuxX.Y` or `linuxX.X`?

Reply:
Fair point - i can't see any reason *not* to change it to `X.Y`, so i'll do that.